src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/share/model/SpeakerRequestBuildContext.java

```diff
@@ -25,10 +25,28 @@
 @AllArgsConstructor
 @EqualsAndHashCode
 public class SpeakerRequestBuildContext {
-    private boolean removeCustomerPortRule;
-    private boolean removeOppositeCustomerPortRule;
-    private boolean removeCustomerPortLldpRule;
-    private boolean removeOppositeCustomerPortLldpRule;
-    private boolean removeCustomerPortArpRule;
-    private boolean removeOppositeCustomerPortArpRule;
+    public static final SpeakerRequestBuildContext EMPTY = SpeakerRequestBuildContext.builder()
+            .forward(PathContext.builder().build())
+            .reverse(PathContext.builder().build())
+            .build();
+
+    private PathContext forward;
+    private PathContext reverse;
+
+    /**
+     * Swap forward and reverse PathContext.
+     */
+    public void swap() {
```

Comment:
I am not sure if this is a good idea. It is easy to forget in what condition we were and in what condition we will be after this operation. For me, an explicit constructor is better. And instead of this method, call the constructor to get a new instance.

Reply:
okay